.github/workflows/release-build.yml

```diff
@@ -69,7 +69,7 @@ jobs:
 
       # https://github.com/sigstore/cosign-installer
       - name: Install Cosign
-        uses: sigstore/cosign-installer@main
+        uses: sigstore/cosign-installer@v1
```

Comment:
@saschagrunert how does the `cosign-release: 'v1.4.1'` down below work with this? Any recommendation what version should we target here?

Reply:
I would assume that v1 installs the latest cosign v1.x.y. Orthogonal to this I would expect for v2 when cosign v2.0.0 got released.

Reply:
So do you think it is safe to remove that specific version?

Reply:
I have checked sigstore/cosign-installer repo and there is a v2.4 released. Maybe we should update it?

Reply:
Ugh, maybe I was totally wrong. Will there ever be a v2 of that action? Who knows 💁🏻‍♂️

Reply:
the tag is wrong, will fix that

Reply:
fixed, now there is a `v2` tag
https://github.com/kedacore/keda/pull/3187